salt/utils/sdb.py

```diff
@@ -57,7 +57,13 @@ def sdb_get(uri, opts, utils=None, strict=False):
     query = uri[indx + 1 :]
 
     loaded_db = salt.loader.sdb(opts, fun, utils=utils)
-    return loaded_db[fun](query, profile=profile)
+    sdb_value = loaded_db[fun](query, profile=profile)
```

Comment:
Why is it not always returning a string? My concerns is there might be use cases where someone would want a bytes value.